cmd/flightctl-api/main.go

```diff
@@ -110,22 +114,29 @@ func main() {
 			log.Fatalf("creating listener: %s", err)
 		}
 
-		agentserver := agentserver.New(log, cfg, store, ca, listener)
+		agentserver := agentserver.New(log, cfg, store, ca, listener, metrics)
 		if err := agentserver.Run(ctx); err != nil {
 			log.Fatalf("Error running server: %s", err)
 		}
 		cancel()
 	}()
 
 	go func() {
-
 		grpcServer := agentserver.NewAgentGrpcServer(log, cfg, grpcTlsConfig)
 		if err := grpcServer.Run(ctx); err != nil {
 			log.Fatalf("Error running server: %s", err)
 		}
 		cancel()
 	}()
 
+	go func() {
+		metricsServer := instrumentation.NewMetricsServer(log, cfg, metrics)
+		if err := metricsServer.Run(ctx); err != nil {
+			log.Fatalf("Error running server: %s", err)
```

Comment:
I am surprised this compiles I think this needs to be
```
			log.Fatalf("Error running server: %v", err)
```

Reply:
The `%s` is all over the source code:

```
[cloud-user@MOC-R4PAC24U35-S3B flightctl]$ grep "running server: %s" -r
cmd/flightctl-periodic/main.go:         log.Fatalf("Error running server: %s", err)
cmd/flightctl-worker/main.go:           log.Fatalf("Error running server: %s", err)
cmd/flightctl-api/main.go:                      log.Fatalf("Error running server: %s", err)
cmd/flightctl-api/main.go:                      log.Fatalf("Error running server: %s", err)
cmd/flightctl-api/main.go:                      log.Fatalf("Error running server: %s", err)
cmd/flightctl-api/main.go:                      log.Fatalf("Error running server: %s", err)
```
I can put in a NO-ISSUE pr for it if you'd like.

Reply:
nmd I see it calls Stringer under the hood